### PR TITLE
Update infinite-scroll.ts

### PR DESCRIPTION
**Pull request by istiti**
import only subscription

### CODE REVIEW

File: src/modules/infinite-scroll.ts [hunk 1]
```diff
@@ -7,7 +7,8 @@ import {
 import { PositionResolverFactory } from '../services/position-resolver';
 import { ScrollRegister, ScrollRegisterConfig } from '../services/scroll-register';
 import { ScrollResolver } from '../services/scroll-resolver';
-import { Subscription } from 'rxjs/Rx';
+import 'rxjs/Subscription';
```
Comment (orizens) [inline src/modules/infinite-scroll.ts]: something strange happens with this.
The code editor (vscode) expects the "Subscription" object as an interface for the "disposeScroller" property, however, nc-cli compiles successfully.

File: src/modules/infinite-scroll.ts [hunk 2]
```diff
@@ -7,7 +7,8 @@ import {
...
+import 'rxjs/add/operator/mergeMap'
```
Comment (orizens) [inline src/modules/infinite-scroll.ts]: no need for the mergeMap as it is used in scroll register file.

**Comment by orizens** [comment]
@istiti  thanks for pointing it out.
please test beat version and let me know if that fixes the bundle size:
```npm i ngx-infinite-scroll@0.4.1-0```

File: src/modules/infinite-scroll.ts (hunk 1 above)
Comment (istiti) [inline src/modules/infinite-scroll.ts]: donno but I think import rxjs/Rx is bad. It's maybe not due to ngx-infinite-scroll but I don't understand why I get two times rxjs import in vendor and pollyfils bundle [see screenshot](https://cloud.githubusercontent.com/assets/2789203/25167641/8483fc76-24e0-11e7-8b8c-ba83a7e0db67.JPG)

**Comment by orizens** [comment]
closing this as the suggested fix has been adopted to version 0.5.0 as part of a refactor to the whole repo.
thasnk @istiti

**Pull request closed without merging**